### PR TITLE
Bring back Playground.Mac and migrate it to net6

### PR DESCRIPTION
**Pull request by snechaev**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
This PR brings back the Playground.Mac project removed during net6 migration in a f76f09fa8b7b379d43195bfa56818a7500ff7948 commit. And then migrates the Playground.Mac project to the net6.

### :boom: Does this PR introduce a breaking change?
No

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

**Comment by snechaev** [comment]
I'm looking into the cake build problems (VS build looks good at the same time).

**Comment by snechaev** [comment]
Note to reviewers: It is easier to review this PR commit-by-commit. Moreover, the really important changes are only contained in the single small commit - 3f301d703c4ddb2eac274629140f881dd128b3b8

File: Projects/Playground/Playground.Mac/ChildView.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using MvvmCross.Platforms.Mac.Views;
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    [MvxContentPresentation]
+    public partial class ChildView : MvxViewController<ChildViewModel>
+    {
+        public ChildView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/ChildView.cs:14]:
```suggestion
        public ChildView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/ModalView.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+using MvvmCross.Platforms.Mac.Views;
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    [MvxModalPresentation]
+    public partial class ModalView : MvxViewController<ModalViewModel>
+    {
+        public ModalView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/ModalView.cs:14]:
```suggestion
        public ModalView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/RootView.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    [MvxFromStoryboard("Main")]
+    [MvxWindowPresentation(PositionX = 300)]
+    public partial class RootView : MvxViewController<RootViewModel>, IMvxOverridePresentationAttribute
+    {
+        // prevents presentation in a new window when navigating back to root from a child
+        private static bool WasPresentedInWindow = false;
+
+        public bool MyValue { get; set; } = true;
+
+        public RootView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/RootView.cs:24]:
```suggestion
        public RootView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/SheetView.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+using MvvmCross.Platforms.Mac.Views;
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    [MvxSheetPresentation]
+    public partial class SheetView : MvxViewController<SheetViewModel>
+    {
+        public SheetView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/SheetView.cs:14]:
```suggestion
        public SheetView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/Tab1View.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+using MvvmCross.Platforms.Mac.Views;
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    [MvxTabPresentation(TabTitle = "Tab1")]
+    public partial class Tab1View : MvxViewController<Tab1ViewModel>
+    {
+        public Tab1View(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/Tab1View.cs:14]:
```suggestion
        public Tab1View(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/TabsRootView.cs [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxWindowPresentation(PositionX = 150)]
+    public partial class TabsRootView : MvxTabViewController<TabsRootViewModel>
+    {
+        private bool _firstTime = true;
+
+        public TabsRootView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/TabsRootView.cs:13]:
```suggestion
        public TabsRootView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/ToolbarWindow.cs [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+using MvvmCross.Platforms.Mac.Views;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    public partial class ToolbarWindow : MvxWindowController
+    {
+        private static int _count;
+
+        public ToolbarWindow(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/ToolbarWindow.cs:14]:
```suggestion
        public ToolbarWindow(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/WindowView.cs [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+using MvvmCross.Platforms.Mac.Views;
+using Playground.Core.ViewModels;
+
+namespace Playground.Mac
+{
+    [MvxFromStoryboard("Main")]
+    [MvxWindowPresentation("ToolbarWindow", "Main", Width = 500)]
+    public partial class WindowView : MvxViewController<WindowViewModel>
+    {
+        public WindowView(IntPtr handle) : base(handle)
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/WindowView.cs:14]:
```suggestion
        public WindowView(NativeHandle handle) : base(handle)
```

File: Projects/Playground/Playground.Mac/ToolbarWindow.cs [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            _count++;
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+        }
```
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/ToolbarWindow.cs:22]:
```suggestion
```

**Review by Cheesebaron — CHANGES_REQUESTED** [review]

**Comment by Cheesebaron** [comment]
Please add the project to the -macos.slnf solution filter.

File: Projects/Playground/Playground.Mac/Playground.Mac.sln
Comment (Cheesebaron) [inline Projects/Playground/Playground.Mac/Playground.Mac.sln:1]: With the -macos.slnf solution filter, I don't find a separate solution worth having. Please remove.

**Comment by Cheesebaron** [comment]
Looks good to me. Thanks!

**Pull request merged**